node-graph/gcore/src/context.rs

```diff
@@ -348,7 +348,7 @@ impl OwnedContextImpl {
 
 #[derive(Default, Clone, Copy, dyn_any::DynAny)]
 pub struct ContextImpl<'a> {
-	pub(crate) footprint: Option<&'a crate::transform::Footprint>,
+	pub(crate) footprint: Option<&'a Footprint>,
 	varargs: Option<&'a [DynRef<'a>]>,
 	// This could be converted into a single enum to save extra bytes
```

Comment:
Only the owned context should be moved out of the context file, the rest should if possible remain no std

Reply:
This is basically a future note to you that you will need to split this file into two versions, one which uses std (and containts the `OwnedContextImpl`) and a second one which contains the rest so you'll have to change the paths you've converted to `std` in this file for this  pr back to `core`